platform/broadcom/sonic-platform-modules-alphanetworks/snj60b0-320f/sonic_platform/thermal_manager.py

```diff
@@ -0,0 +1,23 @@
+from sonic_platform_base.sonic_thermal_control.thermal_manager_base import ThermalManagerBase
+from .thermal_infos import * # lgtm[py/polluting-import]
+from .thermal_conditions import * # lgtm[py/polluting-import]
+from .thermal_actions import * # lgtm[py/polluting-import]
```

Comment:
Rather than suppressing the alerts, why not import the modules themselves? E.g., 

```
from . import thermal_infos
from . import thermal_conditions
from . import thermal_actions
```

Reply:
[LGTM] : I suppress the 3 import * warning. Reference: #4677.
Those alerts are from Mellanox's codebase.
Mellanox suppress the 3 import * warning too.

Reply:
If follow the suggestion, new alerts will occur.

![alert](https://user-images.githubusercontent.com/37563687/90727751-58ed4b80-e2f6-11ea-9126-ce270ae7ac95.png)

Reply:
Currently I follow Mellanox to suppress the 3 import * warning. Reference: #4677.

Reply:
You get the new alerts because you have to make further modifications to reference the module names in the remainder of the file. I will accept this for now since it is done this way in the Mellanox implementation, but it should be refactored in the future.